src/analysis/special_functions/log.lean

```diff
@@ -207,6 +207,18 @@ begin
     (h.tendsto.mono_left inf_le_left)
 end
 
+open_locale big_operators
+
+lemma sum_log_eq_log_prod {α : Type*} (s : finset α) (f : α → ℝ) (hf : ∀ x ∈ s, f x ≠ 0):
```

Comment:
In analogy with monoid_homs, I would expect this in the other direction as `log_prod`.
Also, can't we make this a 1-line proof using some sort of `log_hom` that is a monoid_hom to `multiplicative ℝ`?

Reply:
Good point! I'll amend. I don't think this can be a 1-line proof because of the non-zero condition?